### ISSUE TITLE
add title to alt text for 'View Availability' and 'Bookmark' in search results

### ISSUE THREAD

**Issue by Smullz622** [psu-libraries/psulib_blacklight#1391]
In search results, there is a 'View Availability' and 'Bookmark' button for each result. It's more obvious which record they refer to when tabbing through, but when links on the page are listed, they all just say the same thing. We want the alt text for these buttons to include the title they are associated with.

**Comment by granta03** [comment]
@Smullz622 when I try to tab through the links on VoiceOver, it skips the 'bookmark' button entirely. When links are listed, neither the 'view availability' nor the 'bookmark' buttons show up either. On VoiceOver, you can navigate through the page in two different ways. Using the tab key only goes through what should be hyperlinks and using Ctrl-Option-left or right arrow goes through more elements. Figured I should point that out since I'm a little confused what you mean by tabbing.

**Comment by Smullz622** [comment]
Sorry, that's my incorrect wording choice. I think it may be tab in other screen reader programs. Essentially what I meant was just that when navigating through it's easier to tell which record buttons are associated with as they go in order, but when links/buttons are listed, it's unclear because it's just 10 links that all say 'View Availability'. That must just a difference with VoiceOver that it's not listing 'View Availability' or 'Bookmark' in links, but it did on JAWS with Michelle.
